src/containers/MainContainer/MainContainer.js

```diff
@@ -38,28 +40,21 @@ const MainContainer = () => {
         localStorage.setItem('preferredLanguage', language);
     };
 
-    useEffect(() => {
-        const script = document.createElement('script');
-        script.src = 'https://js.hsforms.net/forms/shell.js';
-        document.body.appendChild(script);
-
-        script.addEventListener('load', () => {
-            if (window.hbspt) {
-                window.hbspt.forms.create({
-                    portalId: '8034478',
-                    formId: content[language].hubspot,
-                    target: `#hubspotForm-en`,
-                });
-            }
-        });
-    }, [language]);
     const videoEndedHandler = (event) => {
         setVideoState({
             hasWatchedVideo: true,
         });
         nextStep();
     };
 
+    useEffect(() => {
+        axios
+            .get('http://localhost:5000/api/questionnaires')
```

Comment:
need to move this to URLs.js i think

Reply:
I dont think we have a URLs.js file, but we have an apis.js file, so I moved the url to there. Is that kinda what you were thinking or should I create a URLs.js file and move it there?